examples/bls12-381-hash/src/bls12-381-hash.rs

```diff
@@ -94,7 +94,8 @@ pub fn fp_hash_to_field(msg: &ByteSeq, dst: &ByteSeq, count: usize) -> Seq<Fp> {
         let elm_offset = L * i; // L * (j + i * m)
         let tv = uniform_bytes.slice(elm_offset, L); //substr(uniform_bytes, elm_offset, L)
         let u_i =
-            Fp::from_byte_seq_be(&FpHash::from_byte_seq_be(&tv).to_byte_seq_be().slice(16, 48)); // OS2IP(tv) mod p
+            Fp::from_byte_seq_be(&tv.slice(16, 48)); // OS2IP(tv) mod p
+            // Fp::from_byte_seq_be(&FpHash::from_byte_seq_be(&tv).to_byte_seq_be().slice(16, 48)); // OS2IP(tv) mod p
```

Comment:
It was not a mistake to do from_byte_seq followed by to_byte_seq. In between nat_mod reduces it mod p.

Reply:
It seems this information is not propagated to Coq, so I will add that, and read the from_seq, to_seq. Otherwise I might just split it into two lines, so Coq gets the correct type information.